docker/entrypoint.sh

```diff
@@ -23,6 +23,7 @@ mkdir -p /app/configs/chains/C
 
 cat <<EOF > /app/configs/chains/C/config.json
 {
+  "state-sync-enabled": false,
```

Comment:
This looks right to me to explicitly disable state sync

Reply:
For context, we are changing the default state sync behavior so that state sync is enabled by default for any node that is starting from scratch.

https://github.com/ava-labs/coreth-internal/pull/1178